<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RC522 Reader</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查找USB设备</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>531</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>设备信息</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>401</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>设 备 名 称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="DeviceName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>401</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>厂       商：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Vender">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>401</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>序   列  号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Serial">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>401</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>厂   商  ID：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="VenderID">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>401</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>产   品  ID：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ProductID">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>401</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>USB协议版本：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="USB">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>LED及按键控制</string>
    </attribute>
    <widget class="QCheckBox" name="LED1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>109</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Aparajita</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>LED1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>501</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>基本操作</string>
    </attribute>
    <widget class="QTextBrowser" name="base_msg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>511</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>501</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="REQA">
        <property name="text">
         <string>寻卡</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AnticollA">
        <property name="text">
         <string>防冲突</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="select">
        <property name="text">
         <string>选择</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="halta">
        <property name="text">
         <string>终止</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>读写操作</string>
    </attribute>
    <widget class="QTextBrowser" name="rw">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>501</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>11</y>
       <width>491</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLineEdit" name="key"/>
      </item>
      <item>
       <widget class="QRadioButton" name="key4a">
        <property name="text">
         <string>A组密码</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="key4b">
        <property name="text">
         <string>B组密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Verify">
        <property name="text">
         <string>验证密匙</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>78</y>
       <width>391</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>块1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>134</y>
       <width>391</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>块3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>106</y>
       <width>391</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>块2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>50</y>
       <width>391</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>块0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>50</y>
       <width>77</width>
       <height>106</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="section">
        <item>
         <property name="text">
          <string>扇区0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区16</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="block">
        <item>
         <property name="text">
          <string>扇区0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>扇区3</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="section_read">
        <property name="text">
         <string>读取</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="section_write">
        <property name="text">
         <string>写入</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="DevOpen">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
